### ISSUE TITLE
Accessibility Audit: Screen readers do not announce the selection of images in the side menu 

### ISSUE THREAD

**Issue by brittnylapierre** [ProjectMirador/mirador#3884]
Description: An accessibility audit of Mirador was performed. Description: It was found that there was not enough auditory feedback for users with screen readers when clicking an image to view. 

Supporting Comments:
- DEAF-BLIND – Windows 10/JAWS 2019/Focus 14 braille/Chrome - Used arrows to navigate through the list of images but JAWS gave no clear indication of movement 
- DEAF-BLIND – Windows 11/laptop/JAWS 2020/PacMate 20 braille/Chrome - Improved experience with JAWS 2020, but still encountering issues with image display and page tracking. 
- LIMITED SIGHT – Windows 10/Windows Narrator/EDGE Narrator did not announce pressed images. 
- LIMITED SIGHT – Windows 10/Windows Narrator/EDGE - Single images read as menu items, but Narrator didn't announce or indicate the selected image. 
- LIMITED SIGHT – Windows 10/Windows Narrator/EDGE - Narrator did not announce pressed images. 
- LIMITED SIGHT – Windows 10/NVDA 2022.4/ Chrome - Images in NVDA appear as menu items; clicking them announces the item but doesn't indicate the selected one. 
- LIMITED SIGHT – iPhone 14 Pro/iOS 17.2/VoiceOver/Safari - Clicking on each image in the Index area doesn't prompt VoiceOver announcements.
